### PR TITLE
IRONdb 1.1.0

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: irondb/release-notes.md [hunk 1]
```diff
@@ -5,6 +5,62 @@ sidebar_position: 12
...
+ * The reconstitute GUI no longer apppears when doing online reconstitutes.
+ * Fix iteration bug when reconstituting NNTBS shards.
+ * Added the `merge_all_nodes` flag to live reconstitute which causes all
+   available and non-blacklisted write copies to send metric data instead of
+   only the "primary" available node.
+ * Added the ability to repair a local database by reconstituting a single
+   metric stream.
+ * Fix bug where `/fetch` would not proxy if the data for a time period was all
+   in the raw database, but the relevant raw shards were offline.
+ * [libmtev 2.5.3](https://github.com/circonus-labs/libmtev/blob/master/ChangeLog.md#253)
```
Comment (pamaddox) [inline irondb/release-notes.md:61]:
```suggestion
```

Comment (pamaddox) [reply irondb/release-notes.md:61]: This was already in the prior release, don't think we need to add it again

Comment (esproul) [reply irondb/release-notes.md:61]: Why remove this?

Comment (esproul) [reply irondb/release-notes.md:61]: Nevermind, I see that we got 2.5.3 with 1.0.1.

**Review by pamaddox — APPROVED** [review]
Looks good, assuming this is the version we go with

**Pull request merged**